models/reviews.model.js

```diff
@@ -71,16 +70,45 @@ const fetchReviewCommentsFromId = (id) => {
     return comments;
   });
 
-  let promises = [returnReviewIfExists, returnCommentsIfExist];
+  const promises = [returnReviewIfExists, returnCommentsIfExist];
 
   return Promise.all(promises).then(([review, comments]) => {
-    if (!review) {
+    return comments;
+  });
+};
+
+const updateCommentToReviewFromId = (id, reqBody) => {
+  const { body, username } = reqBody;
+  if (!body || !username) {
+    return Promise.reject({
+      status: 400,
+      msg: "Bad request.",
+    });
+  }
+
+  const commentQueryStr = `
+  INSERT INTO comments
+  (body, review_id, author)
+  VALUES
+  ($1, $2, $3)
+  RETURNING *;`;
+
+  const reviewExists = fetchReviewById(id).then((review) => review);
+  const userExists = db.query(`SELECT * FROM users WHERE username = $1`, [
+    username,
+  ]);
+
```

Comment:
For checking the review exists, do we need to append the .then to pick up the review and then return it? 

We don't use the review as part of this request, we just invoke the fetchReviewById function so that it will throw an error for us if there is no review there.